app/blocks/input/prefixScript.js

```diff
@@ -33,16 +33,25 @@ class PrefixScript extends InputBlock {
       return false
     }
     var regex = ['^']
-    if (!this.isScoped) {
+    if (this.isScoped) {
+      if (this.args.match(/^r/i)) {
```

Comment:
Exactly. They were all distinct enough this was fine. Do you think it should be more verbose?